prime-angular-frontend/src/app/modules/site-registration/pages/remote-user/remote-user.component.html

```diff
@@ -0,0 +1,105 @@
+<app-page>
+  <app-page-header>PharmaNet Site Registration</app-page-header>
+  <app-site-progress-indicator [inProgress]="!isCompleted"
+                               [currentRoute]="SiteRoutes.REMOTE_USERS"></app-site-progress-indicator>
+
+  <form [formGroup]="form"
+        (ngOnSubmit)="onSubmit()"
+        novalidate>
+
+    <app-page-subheader>
+      <ng-container appPageSubheaderTitle>
+        Remote User
+      </ng-container>
+    </app-page-subheader>
+
+    <section class="mb-3">
+      <div class="row">
+        <div class="col-12">
+
+          <mat-form-field class="w-100">
+            <input matInput
+                   placeholder="First Name"
+                   formControlName="firstName">
+            <mat-error>Required</mat-error>
+          </mat-form-field>
+
+        </div>
+        <div class="col-12">
+
+          <mat-form-field class="w-100">
+            <input matInput
+                   placeholder="Last Name"
+                   formControlName="lastName">
+            <mat-error>Required</mat-error>
+          </mat-form-field>
+
+        </div>
+      </div>
+    </section>
+
+    <app-page-subheader>
+      <ng-container appPageSubheaderTitle>
+        Remote Address
+      </ng-container>
+      <ng-container appPageSubheaderSummary>
+        The remote address is the location the practitioner will be accessing PharmaNet outside the physical boundaries
+        of the authorized community practice location (e.g., from their home). The remote address cannot be another
+        healthcare practice setting.
+      </ng-container>
+    </app-page-subheader>
+
+    <ng-container *ngFor="let remoteUserLocation of remoteUserLocations.controls; let i = index; let last = last"
+                  formArrayName="remoteUserLocations">
+      <ng-container [formGroup]="remoteUserLocation">
+
+        <app-form-icon-group [show]="true"
+                             (event)="onRemove(i)">
+          <app-form-icon-group>
+
+            <section [class.mb-4]="!last">
+              <div class="row">
+                <div class="col-12">
+
+                  <mat-form-field class="w-100">
+                    <input matInput
+                           placeholder="Internet Provider at this location (e.g., Shaw, Telus)"
+                           formControlName="internetProvider">
+                    <mat-error>Required</mat-error>
+                  </mat-form-field>
+
+                </div>
+              </div>
+
+              <app-address [form]="remoteUserLocation.get('physicalAddress')"
+                           [formControlNames]="formControlNames"></app-address>
+
+            </section>
+
+            <ng-container appContextualContent>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Fuga possimus nostrum accusamus doloribus esse
```

Comment:
lorem ipsum surprises me here.

Reply:
yep, I'll check remove context component wrapper, and if they want it back they can provide text.